charts/portworx/templates/portworx-controller.yaml

```diff
@@ -114,8 +116,17 @@ spec:
           httpGet:
             host: 127.0.0.1
             path: /healthz
+            {{- if semverCompare ">=1.22" .Capabilities.KubeVersion.GitVersion }}
+            {{- if $AKSorEKSInstall }}
+            port: {{ add .Values.pxStartPort 30 }}
```

Comment:
I checked the helm chart and looks like we have hard-coded the start port everywhere to 9001. I would recommend not adding the `pxStartPort` parameter right now because it is only going to be used here in the pvc-controller, while the rest of the places are not going to change. Just to avoid confusion lets remove the parameter and hard-code the value to `9031` here too.

Reply:
updated :)